Dockerfile

```diff
@@ -1,25 +1,47 @@
-FROM miyurud/jasminegraph
+FROM ubuntu:focal
+WORKDIR /home/ubuntu
+RUN mkdir software
 WORKDIR /home/ubuntu/software
+
 RUN apt-get update
-COPY . jasminegraph/
+RUN apt-get install --no-install-recommends -y apt-transport-https
+RUN apt-get update
+RUN apt-get install --no-install-recommends -y curl gnupg2 ca-certificates software-properties-common nlohmann-json3-dev
+RUN apt-get install --no-install-recommends -y git cmake build-essential sqlite3 libsqlite3-dev libssl-dev librdkafka-dev libboost-all-dev libtool libxerces-c-dev libflatbuffers-dev python3-pip
+RUN add-apt-repository ppa:deadsnakes/ppa
+RUN apt-get install --no-install-recommends -y python3.5-dev
+RUN apt-get install --no-install-recommends -y libjsoncpp-dev libspdlog-dev pigz
+
+RUN curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -
+RUN add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+RUN apt-get update
+RUN apt-get install --no-install-recommends -y docker-ce-cli
+
+RUN git clone --single-branch --depth 1 https://github.com/chinthakarukshan/metis.git
```

Comment:
Rather than depending on Chinthaka's repo for Metis we should download from the original Metis project and build from those sources.

Reply:
Done